library/core/src/lib.rs

```diff
@@ -119,7 +119,7 @@
 #![feature(nll)]
 #![feature(exhaustive_patterns)]
 #![feature(no_core)]
-#![feature(optin_builtin_traits)]
+#![feature(auto_traits)]
```

Comment:
```suggestion
#![cfg_attr(bootstrap, feature(optin_builtin_traits))]
#![cfg_attr(not(bootstrap), feature(auto_traits))]
```

This needs to be done for core, alloc and std at least, not sure which others, but it does not need to be done for tests or cranelift.

Reply:
Do I need to add that to rtstartup?

Reply:
I added it rtstartup; we'll see if CI passes!